package.json

```diff
@@ -20,12 +20,16 @@
     "prepublish": "grunt test"
   },
   "devDependencies": {
+    "async": "^0.9.0",
```

Comment:
yuck. Okay, filed 2 new issues
https://github.com/freedomjs/freedom-for-firefox
https://github.com/freedomjs/freedom-for-chrome/issues/58

I believe @willscott is taking care of these today and hopefully we can get rid of these dependencies here